report.txt

```diff
@@ -0,0 +1,83 @@
+
+
+Report
+======
+117 statements analysed.
+
+Statistics by type
+------------------
+
++---------+-------+-----------+-----------+------------+---------+
+|type     |number |old number |difference |%documented |%badname |
++=========+=======+===========+===========+============+=========+
+|module   |6      |6          |=          |100.00      |0.00     |
++---------+-------+-----------+-----------+------------+---------+
+|class    |0      |NC         |NC         |0           |0        |
++---------+-------+-----------+-----------+------------+---------+
+|method   |0      |NC         |NC         |0           |0        |
++---------+-------+-----------+-----------+------------+---------+
+|function |0      |NC         |NC         |0           |0        |
++---------+-------+-----------+-----------+------------+---------+
+
+
+
+Raw metrics
+-----------
+
++----------+-------+------+---------+-----------+
+|type      |number |%     |previous |difference |
++==========+=======+======+=========+===========+
+|code      |136    |62.10 |136      |=          |
++----------+-------+------+---------+-----------+
+|docstring |32     |14.61 |32       |=          |
++----------+-------+------+---------+-----------+
+|comment   |12     |5.48  |12       |=          |
++----------+-------+------+---------+-----------+
+|empty     |39     |17.81 |39       |=          |
++----------+-------+------+---------+-----------+
+
+
+
+Duplication
+-----------
+
++-------------------------+------+---------+-----------+
+|                         |now   |previous |difference |
++=========================+======+=========+===========+
+|nb duplicated lines      |0     |0        |0          |
++-------------------------+------+---------+-----------+
+|percent duplicated lines |0.000 |0.000    |=          |
```

Comment:
How come there are no duplicates line according to the report, but there are in the actual files

Reply:
it should be fixed now, it only analysis packages with an __init__.py and now they are added so it should analyse everything. Can you double check if this works for you now?

Reply:
It seems to work fine for me now